tasks/section1.yml

```diff
@@ -477,7 +477,7 @@
       - rule_1.1.14
 
 - name: "SCORED | 1.1.15 | AUDIT | Ensure nodev option set on /dev/shm partition"
-  command: /bin/true
+  shell: mount | grep /dev/shm | grep nodev
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*